docs/openapi/components/schemas/common/LEIentity.yml

```diff
@@ -152,7 +152,8 @@ example: |-
     "legalAddress": {
       "type": ["LEIaddress"],
       "language": "fa",
```

Comment:
Nothing I can see is in [Persian](https://localizely.com/language-code/fa/)...

Reply:
fixed